static/js/activity.js

```diff
@@ -320,23 +321,86 @@ exports.insert_user_into_list = function (user_id) {
     compose_fade.update_one_user_row(elt);
 };
 
+function get_users(search_term) {
+    var all_persons = [];
+    if (search_term.indexOf('') !== -1) {
```

Comment:
Sorry, I should change search_term to search_terms. I'm passing in an array not just a string. Also this was what was causing the casper issues before - array.includes is an ES6 feature